pkg/kubelet/kubelet.go

```diff
@@ -1741,6 +1777,11 @@ func (kl *Kubelet) generatePodStatus(pod *api.Pod) (api.PodStatus, error) {
 		podStatus.HostIP = hostIP.String()
 	}
 
+	podStatus.StartTime = pod.Status.StartTime
```

Comment:
Can we consolidate resetting StartTime logic into one place so that it is not that fragile due to lack of checkpointing in kubelet today? How about moving it together with SetPodStatus in status_manager?